composer.lock

```diff
@@ -1653,16 +1653,16 @@
         },
         {
             "name": "christophwurst/kitinerary",
-            "version": "0.2.2",
+            "version": "0.2.3",
             "source": {
                 "type": "git",
                 "url": "https://github.com/ChristophWurst/kitinerary.git",
-                "reference": "5e0f19c16af95821d85f4a8da72e571ccc818601"
+                "reference": "b3fb70ec4351af9aaa4999b1dd8deb26caeccd20"
```

Comment:
revert please

bump `php` only (`composer update php`) and keep the other package untouched